src/screens/ContactInformation.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+import {NEXT_MESSAGE} from '../utils/Constants';
+import {Colors} from '../utils/Colors';
+import DescContactInfo from '../components/DescContactInfo';
+import EmailPassTxtInput from '../components/EmailPassTxtInput';
+
+const ContactInformation = () => {
+  const titleInput = ['Nombre', 'Teléfono'];
+
+  const putInputs = () => {
+    const inputs = titleInput.map((element) => {
+      if (element === 'Nombre') {
+        return (
+          <EmailPassTxtInput
+            key={element}
+            id={element}
+            title={element}
+            isRequired="Recomendado"
```

Comment:
```suggestion
            isRequired='Recomendado'
```

change all double-quotes for simple-quotes on the files